packages/utils/src/index.ts

```diff
@@ -256,3 +257,43 @@ export const humanize = (str: string) => {
     .map(upcase)
     .join(' ');
 };
+
+export const getImageRef = (mode: ImageMode, id: string, extension: ImageExtension) =>
+  `${mode}:${id}.${extension}`;
+
+export const isValidImageRef = (ref: string): boolean => {
+  if (/^(local):(.+)\.(gif|jpg|png|webp)$/.test(ref)) {
+    return true;
+  }
+
+  return false;
+};
+
+export const SUPPORTED_IMAGE_EXTENSIONS = ['jpg', 'png', 'webp', 'gif'];
+
+const isValidImageExtension = (extension: string): boolean =>
+  SUPPORTED_IMAGE_EXTENSIONS.includes(extension);
+
+export const parseImageRef = (
+  ref: string
+): { mode: ImageMode; id: string; extension: ImageExtension } => {
+  const throwInvalidRefError = () => {
+    throw new Error('Invalid image reference');
+  };
+  const [mode, idAndExt] = ref.split(':');
+  const [id, ext] = idAndExt.split('.');
```

Comment:
This isn't right, this won't support file names with `.` in it. You should match on the regex here